RecoBTag/PerformanceDB/test/TestBtagPayloads.cc

```diff
@@ -98,48 +168,11 @@ void TestBtagPayloads::analyze(const edm::Event& iEvent, const edm::EventSetup&
 
   printf("\033[22;31m TESTING FOR TTBAR WP's \n\033[0m");
 
-  //Possible algorithms: TTBARWPBTAGCSVL,  TTBARWPBTAGCSVM,   TTBARWPBTAGCSVT
-  //                     TTBARWPBTAGJPL,   TTBARWPBTAGJPM,    TTBARWPBTAGJPT
-  //                                                          TTBARWPBTAGTCHPT
```

Comment:
```suggestion
  //Possible algorithms: TTBARWPBTAGCSVL,  TTBARWPBTAGCSVM,   TTBARWPBTAGCSVT
  //                     TTBARWPBTAGJPL,   TTBARWPBTAGJPM,    TTBARWPBTAGJPT
  //                                                          TTBARWPBTAGTCHPT
```
this comment is lost in full; perhaps instead of recovering here, better copy this up above before filling the `measureName_`

Reply:
Good catch.